hpcstats.spec

```diff
@@ -0,0 +1,122 @@
+Name:		hpcstats		
+Version:	1.6.3	
+Release:	1%{?dist}.edf
+Summary:	HPC cluster usage accounting and reporting software	
+
+License:	Proprietary
+URL:		http://github.com/edf-hpc/hpcstats
+Source0:	%{name}-%{version}.tar.gz
+
+BuildRequires:	python3-setuptools
+BuildRequires:	python3-sphinx
+BuildRequires:	python3-sphinx_rtd_theme
+Requires: rpm-cron
+Requires: postgresql
+
+%description
+HPC cluster usage accounting and reporting software 
+HPCStats imports raw data from various sources (Job schedulers, log
+files, LDAP directories, etc. and insert everything into a coherent
+and structured PostgreSQL database. Then, all these structured
+information can be used to extract statistics about the usage of the HPC clusters.
+
+%prep
+%global debug_package %{nil}
+%setup -q
+
+%build
+CFLAGS="%{optflags}" python3 setup.py build
+
+%install
+install -d %{buildroot}%{_datadir}/%{name}
+install -d %{buildroot}%{_sysconfdir}/%{name}
+install -d %{buildroot}%{_datadir}/%{name}/bin
+install -d %{buildroot}/%{_datadir}/%{name}/db
```

Comment:
general "style" comment (that is not worth fixing overall imho):
choose if you want to append a / between the folder variables or not.
ie: %{buildroot}__/__%{_datadir} or %{buildroot}%{_datadir}